spec/acceptance/unbound_spec.rb

```diff
@@ -15,7 +15,7 @@
 describe 'unbound class' do
   describe 'running puppet code' do
     it 'work with no errors' do
-      pp = "class {'unbound': }"
+      pp = "class {'unbound': verbosity => 5, logfile => '/tmp/unbound.log'}"
```

Comment:
If this happens more often, https://github.com/voxpupuli/voxpupuli-acceptance/#environment-variables-to-facts may be a good way to make this easier.

Reply:
this was mainly to debug the ipv6 issue and not sure if its genrally usefull but thanks for the pointer